tests/testthat/test-check_pkg_installed.R

```diff
@@ -21,16 +21,18 @@ test_that("check_pkg_installed() works", {
       compare = ">="
     )
   )
+
+  # checking that the function still works when there is no reference package.
   expect_equal(
-    get_min_version_required("brms", reference_pkg = NULL),
```

Comment:
I think we can delete this testing file altogether. I think it's leftover from when these functions were exported from cards

Reply:
good point I've deleted the file!

Reply:
great, thank you! sorry i didn't realize when we were looking at this file earlier haha